packages/apps-routing/src/accounts.ts

```diff
@@ -12,7 +13,7 @@ export default function create (t: <T = string> (key: string, text: string, opti
     display: {
       needsApi: []
     },
-    icon: 'users',
+    icon: Icon as string,
```

Comment:
As explained above, this really has wide-ranging impacts. So would keep to the fa set (imperfect as it may be)